### PR TITLE
check if primitive value

### PR DESCRIPTION
**Pull request by fesebuv**
* Initial Commit
* Includes tests and setup

### CODE REVIEW

File: test/is-primitive.spec.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    it('undefined', function () {
+      expect(isPrimitive(undefined)).to.be.true;
+    });
+
+    it('NaN', function () {
+      expect(isPrimitive(NaN)).to.be.true;
+    });
+
+    it('Infinite', function () {
+      expect(isPrimitive(Infinity)).to.be.true;
```
Comment (pgoldrbx) [inline test/is-primitive.spec.js:33]: I'm not actually sure if `NaN` and `Infinity` are considered primitive.
I think what you want:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures

**update**: I'm wrong!
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Number_type

File: test/is-primitive.spec.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+var expect = require('chai').expect;
```
Comment (pgoldrbx) [inline test/is-primitive.spec.js:3]: `chai` is not listed in the package dependencies

**Review by pgoldrbx — COMMENTED** [review]
one dev dependency missing from `package.json`. otherwise, the readme could include links to relevant documentation

File: test/is-primitive.spec.js (hunk 2 above)
Comment (fesebuv) [reply test/is-primitive.spec.js:3]: thanks for catching that. I've added it to package dependencies.

**Review by pgoldrbx — APPROVED** [review]
looks great!

**Comment by fesebuv** [comment]
@pgoldrbx thanks!

**Pull request merged**